py/reporter_service.py

```diff
@@ -50,6 +51,24 @@ def serve_forever(self):
   def make_thread_locals(self):
     setattr(thread_local, 'segment_matcher', valhalla.SegmentMatcher())
 
+    levels = "0,1"
```

Comment:
these four lines could be just:

`setattr(thread_local, 'report_levels', set([ int(i) for i in os.environ.get('REPORT_LEVELS', '0,1').split(',')]))`

same for the rest of these